src/Carbon/Traits/Localization.php

```diff
@@ -643,7 +643,7 @@ public static function localeHasPeriodSyntax($locale)
         return static::executeWithLocale($locale, function ($newLocale, TranslatorInterface $translator) {
             return $newLocale &&
                 $translator->trans('period_recurrences') !== 'period_recurrences' &&
-                $translator->trans('period_interval') !== 'period_interval' &&
+                $translator->trans('period_interval', ['']) !== 'period_interval' &&
```

Comment:
Is it needed? `':period' => (string) $period,` should convert `null` into `""` already.

Reply:
Checked, it is. I can't see any relevant use case for using translator directly and translate such `period_interval` without parameter, but I will double-check.